CHANGELOG/v1.7.2.md

```diff
@@ -0,0 +1,44 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.26.x -> v1.30.x
+- Workload Cluster: v1.24.x -> v1.30.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Highlights
```

Comment:
As the k8s v1.30 support was already added in CAPI v1.7.1, I guess this can be removed from v1.7.2 changelogs 

https://github.com/kubernetes-sigs/cluster-api/blob/fe5001b4a5ee8e9db4cee70e1754c8c8fd6eac38/CHANGELOG/v1.7.1.md?plain=1#L8-L11

Previous Art:

1.29 support added to 1.6.1
https://github.com/kubernetes-sigs/cluster-api/blob/fe5001b4a5ee8e9db4cee70e1754c8c8fd6eac38/CHANGELOG/v1.6.1.md?plain=1#L8-L11

1.6.2 release changelogs

https://github.com/kubernetes-sigs/cluster-api/blob/fe5001b4a5ee8e9db4cee70e1754c8c8fd6eac38/CHANGELOG/v1.6.2.md?plain=1#L7-L10

Reply:
Updated the PR